scripts/local/run_tests.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/bash
```

Comment:
This seems to cause macos/zsh to complain - `#!/usr/bin/env bash` should cover a wider range of shell setups.